src/server/db/migrations/20200425031150-create-user-searches.js

```diff
@@ -11,23 +11,27 @@ module.exports = {
       zip_code: {
         type: Sequelize.STRING,
       },
-      user_latitude: {
+      latitude: {
         type: Sequelize.FLOAT,
       },
-      user_longitude: {
+      longitude: {
         type: Sequelize.FLOAT,
       },
```

Comment:
Two items for review:

1. We are not currently defining latitude and longitude as `DataTypes.DECIMAL(9,6)` as we are in the tables for test centers. Is this something we should update?

2.. We are capturing latitude and longitude, should we consider adding geolocation to this table, too?

Reply:
@KevinGenus good point on the DECIMAL, will fix that :)

Regarding lat/lng/geolocation, currently there are two queries:
- query by zip (in this case we don't know the user's location)
- query by lat,lng (in this case, lat,lng come directly from the user's geolocation)

In order to track more details about a user sending multiple queries, we'll need some kind of persistent user information - like a cookied ID, or IP, etc. There are some privacy implications here, might be better to launch without that detail of tracking and then add-on soon.